huntsman/utils/hub_utils.py

```diff
@@ -0,0 +1,219 @@
+#A Python interface for the acroname USB Hubs - tested and functional
+
+#Please note that the Yaml config file that monique is working on is not yet finalised and is not accessable in POCS yet
+# so not sure what the best approach is here to retrieve info from the file - this can be updated once this is finished 
+
+import brainstem
+import time
+import yaml
+from brainstem.result import Result
+from time import sleep
+
+
+def connect_to_hub(hub_serial, verbose = False, test = True):
+    
+    """
+    Function: Used to connect to the hub and create stem
+    
+    Inputs:
+        
+        hub_serial: Serial number of the hub
+        
+        test: If true, will flash user LED's to verify that a connection has 
+        been made 
+        
+    Outputs: 
+        
+        stem: Connection to hub    
+    
+    """
+    
+    print('Connecting to hub', hub_serial)
+    
+    stem = brainstem.stem.USBHub3p() 
+
+    result = stem.discoverAndConnect(brainstem.link.Spec.USB, hub_serial)
+
+    if  result == (Result.NO_ERROR):
+        
+        result1 = stem.system.getSerialNumber()
+        
+        print ('Connected to USBStem with serial number:', result1)
+    
+        stem.system.setLED(1)
+        
+        if test == True:
```

Comment:
You can just do `if test` otherwise the preferred is `if test is True`. (See note about renaming `test`)

Reply:
Fixed